### ISSUE TITLE
Fail to reproduce multiple videos on Android

### ISSUE THREAD

**Issue by MateusZitelli** [TheWidlarzGroup/react-native-video#302]
When multiple videos are added (more than 3 in my experience) just one video is reproduced while the others stay paused. Anyone knows how to address that?

**Comment by mattapperson** [comment]
@MateusZitelli can you explain what you mean by "reproduced"

**Comment by MateusZitelli** [comment]
Ops, false cognates mistake. When I say "reproduced" I mean played. The problem happens when I have multiple players on the same view playing different videos.

**Comment by mattapperson** [comment]
So this is because of linux decoder, which may should really only be used for 1 video at a time... any more and your risking issues like this. Just a limitation of the platform as far as I am aware (someone please let me know if my info is outdated, I lean more to iOS dev personally). To do something more would require a VERY custom FFMpeg + OpenGL renderer setup

**Comment by mirceaciu** [comment]
sorry to open this but I want to add something to this topic. This might not be OS related as it reproduces both devices and emulators.

I also run into the issue were I created multiple player on screen but only one player was able to play content. Pressing play on one player would pause the other.

Turns out this was because audio focus was preventing multiple audio source to play in the same time, even when I had all of my player muted!  Adding `disableFocus={true}` fixes the issue , now all players can run in the same time!
